lineman/app/components/thread_page/revision_history_modal/revision_history_modal.coffee

```diff
@@ -0,0 +1,20 @@
+angular.module('loomioApp').factory 'RevisionHistoryModal', ->
+  templateUrl: 'generated/components/thread_page/revision_history_modal/revision_history_modal.html'
+  controller: ($scope, discussion, CurrentUser, Records) ->
+    $scope.discussion = discussion
+    
+    Records.versions.fetchByDiscussion($scope.discussion.key)
+
+    $scope.threadTitle = (version) ->
+      $scope.discussion.attributeForVersion('title', version)
+
+    $scope.threadDescription = (version) ->
+      $scope.discussion.attributeForVersion('description', version)
+
+    $scope.versionIsCurrent = (version) ->
```

Comment:
This'll actually be off if we have a tonne of revisions, since we'll only load a single 'page' of them and end up with records that aren't loaded in the client but are on the server.

I'm tempted to raise the default page size of revisions to 100 or something and call it a good problem to have it we have it.

To raise the default page size:

```
# api_versions_controller
default_page_size
  100
end
```